### DISCUSSION TITLE
Added support for GNOME 42 (Successfully merged)

### DISCUSSION THREAD

**Discussion by Weather-OS** [Weather-OS/Fildem-v2#7] [Announcements]
Currently, Fildem doesn't work properly with GNOME 42. 

I'm currently almost done finishing the support for GNOME 42. Expect a release soon.

**Comment by Weather-OS** [comment]
Good news!

**GNOME 42 Compatibility is finally here!**

Example of Fildem-V2 running in GNOME 42
![Fildem-V2](https://gcdnb.pbrd.co/images/Muo6FTD9Ixvb.png?o=1)

I'll be adding package build file support soon.

**Comment by NicolasThierion** [comment]
Thank you for keeping this extention up to date. Works well on my machine - Manjaro Gnome 42.

**Comment by ghost** [comment]
To be honest, the original extension worked fine on my Gnome 42. What changes exactly did you make?
